capitalize_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require './decorator'
+
+# Class for name captalization
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
+  end
```

Comment:
- [ ] As requested by the previous reviewer, please implement the changes asked by the previous reviewer. what this code do `he correct_name method will return the name contained in @nameable with its first letter capitalized and the rest of the letters converted to lowercase.`
````ruby
 def correct_name
      @nameable.correct_name.capitalize
    end
```

Reply:
Thanks for the feedback Roshan,

The above code is just another way of writing the recommended code by the previous reviewer and I prefer as above because it's neat and works.
Kindly advise if it's a must to change it.

Reply:
Hello @sylvester-wamaya, `the correct_name method will return the name contained in `@nameable` with its first letter capitalized and the rest of the letters converted to lowercase. The name should contain in nameable so I am request you pleaese consider it. Regarding the trimmer decorator ` First it check `if @nameable.correct_name.length > 10` if ture then only it it tream the word which is the best way So I request you.
````
def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
````
`

Reply:
Well noted,🙏